sdk/storage/azure-storage-blob/azure/storage/blob/_generated/operations/_block_blob_operations.py

```diff
@@ -889,6 +893,9 @@ def put_blob_from_url(  # pylint: disable=inconsistent-return-statements
         :param copy_source_authorization: Only Bearer type is supported. Credentials should be a valid
          OAuth access token to copy source. Default value is None.
         :type copy_source_authorization: str
+        :param copy_source_tags: Optional, default 'replace'.  Indicates if source tags should be
```

Comment:
Same nit: Extra whitespace

Reply:
Same comment as above, this is generated.